.github/workflows/build-python.yaml

```diff
@@ -57,12 +63,17 @@ jobs:
           repository-name: hawk-ai-aml/${{ inputs.component }}
           repository-ref: ${{ env.GITHUB_REF_NAME }}
           repository-access-token: ${{ secrets.REPO_ACCESS_PAT }}
+          use-sonarcloud: ${{ inputs.useSonarCloud }}
 
   build:
     needs: init
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          # From SonarCloud Scan GitHub Action
+          # Disabling shallow clone is recommended for improving relevancy of reporting
+          fetch-depth: 30
```

Comment:
In Java, we go for 1000 and cloning is pretty fast. We can go with 1000 here too. The more history we have the more accurate result we get.

Reply:
will do in the next PR